apps/ui/components/common/Navigation.tsx

```diff
@@ -157,8 +190,12 @@ function ActionBar({ apiBaseUrl, profileLink }: HeaderProps) {
       logo={<Logo size="large" />}
       logoAriaLabel={`${t("common:applicationName")} logo`}
       logoHref="/"
```

Comment:
I'd use `logoHref={env.NEXT_PUBLIC_BASE_URL}` here since it's possible to change the base url